code/modules/mob/living/carbon/xenomorph/life.dm

```diff
@@ -141,10 +167,15 @@
 		plasma_stored -= 5
 	if(plasma_stored == plasma_max)
 		return
+	var/modifier = 1
+	if(stealth && last_move_intent > world.time - 20) //Stealth halves the rate of plasma recovery on weeds, and eliminates it entirely while moving
```

Comment:
This logic can be reworked to allow for a ` /mob/living/carbon/Xenomorph/Hunter/proc/handle_living_plasma_updates()`

Reply:
This has been done